tasks.py

```diff
@@ -830,6 +830,7 @@ class TestDependencies(NamedTuple):
 
 
 MARKER_DEPENDENCY_MAP: Final[Mapping[str, TestDependencies]] = {
+    "airflow": TestDependencies(("reqs/requirements-dev-airflow.txt",)),
```

Comment:
If you want the test runner to startup your airflow service you need to specify that here.
```suggestion
    "airflow": TestDependencies(("reqs/requirements-dev-airflow.txt",), sevices=("airflow",)),
```